scalableminds/webknossos
# Don't show folders in search results

When a dataset search is active, no folders within the current folder should be shown.

### URL of deployed dev instance (used for testing):
- https://___.webknossos.xyz

### Steps to test:
- go to dashboard table
- create a subfolder in the root (root should stay active)
- use the search bar
- the "you have to enter at least 3 chars" message should appear 
- no folders should be rendered in the search

### Issues:
- fixes #7226
- follow-up for regression caused by #6996 

------
